libraries/EpoxyFS/src/EpoxyFS.h

```diff
@@ -11,6 +11,18 @@
 #include "FS.h"
 #include "FSImpl.h"
 
+// mkdir() is not required by ISO C++, but it is usually available.
+// The catch is that the Unix version has two arguments, while the
+// Windows version has only one - since the Unix-style mode is
+// meaningless on Windows.
+#ifdef __MINGW32__
+#    define sys_mkdir(a, b) mkdir(a)
+// MINGW32 omits lstat because older Windows file systems lack symlinks
+#    define lstat stat
```

Comment:
I would like to avoid clobbering symbols in the global namespace using a `#define`. I've seen `#define` hacks like cause very painful problems in the application code. I think we can use inline functions instead, and even better, move them into the `fs` namespace to avoid any name collision in the global namespace. I'll send you some commits that show this.